pages/docs/web/index.md

```diff
@@ -24,13 +24,12 @@ simply import Capacitor at the top of your app and you're set:
 import { Capacitor } from '@capacitor/core';
 ```
 
-To use a plugin, import `Plugins` and call it, noting that only plugins
-with web support will actually provide useful functionality:
+To use a plugin, install from npm and import the plugin directly:
```

Comment:
this is not the v3 file, the v3 file doesn't include the part that uses the plugin anymore, was removed by dan